src/webserver/www/public/css/game.css

```
@@ -237,6 +238,249 @@ body {
+  height: 32px;
+  padding: 0 12px;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  background: linear-gradient(180deg, #6b8a6b 0%, #5a7a5a 100%);
+  color: #FFF1DA;
+  font-family: 'Comic Relief';
+  font-weight: 600;
+  font-size: 0.9em;
+  border-radius: 5px;
+  border: 2px solid #4a6a4a;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)